### DISCUSSION TITLE
Pregunta sobre los nuevos iconos (o bug visual) para survey builder

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#353] [2. Questions and Answers] [answered]
Hola

Recién me di cuenta en el survey builder cambiaron algunos iconos en la cual veo que confunde, en mi opinión, el significado de ésos.

**Iconos actualmente**: estos se ven en chrome
![image](https://user-images.githubusercontent.com/61419471/126701348-a854c524-083f-429b-836e-8a2f8a07c0f2.png)

**Icono anteriores**: estos se ven en otro navegador (mozilla)
![image](https://user-images.githubusercontent.com/61419471/126701457-d31ef052-6523-48d0-9633-2accfb06e667.png)

O quizás sea algún bug visual? @gmoplaza podrías confirmar plz.

Gracias =>

**Comment by ccarrascor** [comment, marked as answer]
es un bug conocido, al generar los iconos queda cargado en el cache el archivo css anterior. Por ahora, haciendo un hard-reload solucionas esto en tu navegador.

**Reply by NicolasPLG** [reply]
@ccarrascor muchas gracias, haré un hard refresh entonces.
